composer/profiler/dataloader_profiler.py

```diff
@@ -49,4 +49,5 @@ def init(self, state: State, logger: Logger):
                 textwrap.dedent("""To use the dataloader profiler, state.profiler must be set.
                 Make sure to run composer with the profiler -- i.e. with the `--profiler` CLI flag."""))
         state.train_dataloader = ProfiledDataLoader(state.profiler, state.train_dataloader, "train")
-        state.eval_dataloader = ProfiledDataLoader(state.profiler, state.eval_dataloader, "eval")
+        for evaluator in state.evaluators:
+            evaluator.dataloader = ProfiledDataLoader(state.profiler, evaluator.dataloader, "eval")
```

Comment:
I forget how the profiler works,but should the name be the evaluator name rather than just "eval"?